SashimiNinja/.gitignore

```diff
@@ -0,0 +1,4 @@
+node_modules/**/*
```

Comment:
Your folder structure needs adjusted slightly. supporting files to your repo should be on the root level. [Here's some more detail about folder structure](https://www.youtube.com/watch?v=CmTOW-29hpQ&index=2&list=PLEBnLDjR5B912lpuj_8kkqx8zxiiXQB61)